README.rst

```diff
@@ -685,7 +686,8 @@ For convenience, here is a simple example:
           category = 'net',
 
           depends = FUN.autodeps(bin_path) + [ 'bash',
-            'libnew' if VAR.lsb_release == 'trusty' else 'libold' ],
+            'libnew' if VAR.lsb_release == 'trusty' else 'libold' ]
```

Comment:
I prefer to keep the trailing comma, is legal in Python and it makes adding more keys to the dict in the future more hassle-free.